ui/src/components/Terminal/index.vue

```diff
@@ -150,9 +150,27 @@ onMounted(() => {
   :deep(.terminal) {
     height: 100%;
     padding: 10px 0 5px 10px;
+  }
+
+  :deep(.xterm-viewport) {
+    background-color: #000000;
+
+    &::-webkit-scrollbar {
+      height: 4px;
+      width: 7px;
+    }
+
+    &::-webkit-scrollbar-track {
+      background: #000000;
+    }
+
+    &::-webkit-scrollbar-thumb {
+      background: rgba(255, 255, 255, 0.35);
+      border-radius: 3px;
+    }
 
-    .xterm-viewport {
-      overflow-y: unset !important;
+    &::-webkit-scrollbar-thumb:hover {
+      background: rgba(255, 255, 255, 0.5);
     }
   }
 }
```

Comment:
The provided code appears to be incomplete. Could you provide the complete code or specify what aspects need scrutiny? I'll then analyze those specific differences and offer relevant suggestions.

For more comprehensive analysis, please share the actual code snippet.